x/stablestake/module.go

```diff
@@ -120,7 +120,7 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	types.RegisterMsgServer(cfg.MsgServer(), keeper.NewMsgServerImpl(am.keeper))
 	types.RegisterQueryServer(cfg.QueryServer(), am.keeper)
 	m := migrations.NewMigrator(am.keeper)
-	err := cfg.RegisterMigration(types.ModuleName, 8, m.V9Migration)
+	err := cfg.RegisterMigration(types.ModuleName, 9, m.V10Migration)
```

Comment:
not to remove v9 migration, it won't go on mainnet otherwise

Reply:
yeah I asked same, I was wondering how do we update the testnet state now ? main net state update is not required